File: LeetCode Solutions/Arrays 101/Max Consecutive Ones/Max Consecutive Ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        countOfOne = 0; 
        maximumCountOfOne = 0;
        for digit in nums:
            if (digit == 1):
                countOfOne += 1
                maximumCountOfOne = max(maximumCountOfOne, countOfOne)
            else:
                countOfOne = 0
        return maximumCountOfOne

File: LeetCode Solutions/Arrays 101/Max Consecutive Ones/Max Consecutive Ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int countOfOne = 0, maximumCountOfOne = 0;
        for (int digit : nums){
            if (digit == 1){
                countOfOne += 1;
                maximumCountOfOne = Math.max(maximumCountOfOne, countOfOne);
            } else{
                countOfOne = 0;
            }
        }
        return maximumCountOfOne;
    }
}